model/src/main/java/net/spals/oembed4j/model/OEmbedResponse.java

```diff
@@ -96,7 +97,14 @@ public Builder putCustomProperties(final String key, final Object value) {
         }
 
         @Override
+        public Builder setVersion(String version) {
+            Preconditions.checkState(DEFAULT_VERSION.equals(version), "Per the oEmbed spec, versions should always be %s", DEFAULT_VERSION);
+            return super.setVersion(version);
+        }
+
+        @Override
         public OEmbedResponse build() {
+            super.setVersion(DEFAULT_VERSION);
```

Comment:
I think it's more correct to put this in the constructor since there could be a time when more than one version is accepted.